app/views/standup_meeting_groups/index.html.erb

```diff
@@ -1,13 +1,22 @@
-<h1>Standup Meeting Groups</h1>
-
 <% if policy(:standup_meeting_group).new? %>
   <div class="p-2 border-black border-2" id="new_standup_meeting_group">
     <%= render 'form', standup_meeting_group: StandupMeetingGroup.new %>
   </div>
 <% end %>
 
-<hr class="mb-5 mt-5 border-b-2">
+<div class="flex flex-col justify-center">
+  <h2 class="font-bold text-3xl pl-5 text-center md:text-left">My Standups</h2>
+  <div id="my-standup-meeting-groups" class=" flex justify-center grid sm:grid-cols-1 md:grid-cols-2 lg:grid-cols-3 xl:grid-cols-4 gap-5 pl-5 pt-5 pb-5">
```

Comment:
`grid` and `flex` are both `display` properties. They can't be active at the same time. `grid` takes over here as the last property listed but it's not a good idea to leave superfluous properties laying around.

Also, there's quite a bit happening with managing the grid across different viewport sizes. I wonder if you could get the kind of behavior you're after by getting rid of `grid` and just using `flex flex-wrap`.

Reply:
yea, that works much better. Thank you!